client/app/services/events.js

```diff
@@ -1,13 +1,38 @@
-import recordEvent from '@/lib/recordEvent';
+import { debounce, extend } from 'lodash';
+import { $http } from '@/services/ng';
 
-function Events() {
+export let Events = null; // eslint-disable-line import/no-mutable-exports
+
+let events = [];
+
+const sendEvents = debounce(() => {
+  const eventsToSend = events;
+  events = [];
+
+  $http.post('api/events', eventsToSend);
+}, 1000);
+
+function EventsService() {
   this.record = (action, objectType, objectId, additionalProperties) => {
-    recordEvent(action, objectType, objectId, additionalProperties);
```

Comment:
Why move it back into `Events`? There is no point in having an object with a single function in it. The idea was to delete this service when we drop Angular and stay only with `recordEvent`.

Reply:
It should be its own file, because eventually there will be no `Events` service. At this point, we can just delete the `events.js` file and not worry about updating imports.

Unlike other services, here the name is different.

Reply:
@arikfr Done